lib/keyverificationsession.cpp

```diff
@@ -379,12 +379,21 @@ void KeyVerificationSession::handleMac(const KeyVerificationMacEvent& event)
         return;
     }
 
+    auto masterKey = m_connection->masterKeyForUser(m_remoteUserId);
```

Comment:
(Note to myself:)

We need to distinguish between device ids and master keys during verification

Reply:
done